api/test-context/src/test_context.rs

```diff
@@ -438,18 +512,29 @@ impl TestContext {
             .unwrap();
     }
 
+    pub async fn get_sequence_number(&self, account: AccountAddress) -> u64 {
+        let account_resource = self
+            .api_get_account_resource(account, "0x1", "account", "Account")
```

Comment:
Do you know why we need to pass "0x1", "account", "Account" to api_get_account_resource? The account resource is always at the same path

Reply:
this api_get_account_resource() is reused in [string_resource_test](https://github.com/aptos-labs/aptos-core/blob/4b3a539062ff8cd3df5d929447fcac0017353b9f/api/src/tests/string_resource_test.rs#L49-52) with different module and names